project/Win32BuildSetup/BuildSetup.bat

```diff
@@ -50,21 +51,33 @@ SET buildbinaryaddons=true
 SET exitcode=0
 SET useshell=rxvt
 SET BRANCH=na
+SET INSTALLER_ONLY=false
+SET EXE_AND_INSTALLER=false
 FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
```

Comment:
this should have a ,%7, %8 I think

Reply:
That would be true if we wanted to support all the different options at the same time. I don't mind putting it in though.

Reply:
I addressed this.